src/pages/StakingPageSingle.svelte

```diff
@@ -319,79 +319,12 @@
         }
       }
 
-      // Action: stake
-      if(action === 'stake') {
-        if(!stakeAmount)
-          return { ...Errors.AMOUNT_IS_NULL, context: {amount: stakeAmount}};
-
-        let res = await safeFlow('approve');
-        console.log('Approve', res);
-        if(res) return res;
-
-        //Double check with user
-        if(!data.exitFeePercentage.eq(0)) {
-          if(!window.confirm(`On exit a ${formatEther(data.exitFeePercentage.mul(100))}% fee will be charged on your principal`)) {
-            displayNotification({
-              message: Errors.USER_REFUSE_EXIT_FEES.message,
-              type: "hint",
-            });
-            return { ...Errors.USER_REFUSE_EXIT_FEES, context: {fee: data.exitFeePercentage}};
-          }
-        }
-      }
-
       return false;
```

Comment:
I am assuming this final `return false` is the base case, so just deleting the stake logic will resolve to an unknown action and therefore false.

Reply:
every reference to action stake was removed from the codebase also